src/NCronJob/Scheduler/QueueWorker.cs

```diff
@@ -119,6 +120,17 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         }
     }
 
+    private void AssertUseNCronJobWasCalled()
+    {
+        if (!missingMethodCalledHandler.UseWasCalled)
+        {
+            throw new InvalidOperationException("""
```

Comment:
Maybe a small test covering this?

Reply:
I will do this in a follow-up PR, as it gets rather big. I have to touch almost every test.
For now it is "just" a warning